modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/extra/remote-app/src/routes/reports/generateReport/index.tsx

```diff
@@ -198,21 +179,25 @@ const GenerateReport = () => {
 					<div className="col">
 						<label>Liferay Branch</label>
 
-						{branches.map((branch: any, index: string) => (
-							<div
-								className="align-items-center d-flex"
-								key={index}
-							>
-								<Form.Checkbox
-									checked={branchesWatch?.includes(branch.id)}
-									id="branchRequest"
-									label={branch.name}
-									name="branchRequest"
-									onChange={onClickBranches}
-									value={branch.id}
-								/>
-							</div>
-						))}
+						{branches.map(
+							(branch: LiferayBranch, index: string) => (
```

Comment:
What kind is LiferayBranch? Please rename this type to LiferayBranchType
```suggestion
							(branch: LiferayBranchType, index: string) => (
```